components/mpas-ocean/bld/build-namelist-section

```diff
@@ -305,6 +312,7 @@ add_default($nl, 'config_land_ice_flux_ISOMIP_gammaT');
 add_default($nl, 'config_land_ice_flux_rms_tidal_velocity');
 add_default($nl, 'config_land_ice_flux_jenkins_heat_transfer_coefficient');
 add_default($nl, 'config_land_ice_flux_jenkins_salt_transfer_coefficient');
+add_default($nl, 'config_subglacial_runoff_mode');
```

Comment:
This option got moved to the `forcing` namelist group in the registry so it should be moved up here as well.

Reply:
done